submissions/Westerinoo/HTML-CSS-popup/style.css

```diff
@@ -0,0 +1,156 @@
+*{
+    box-sizing: border-box;
+    list-style: none;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+}
+/*-------------------------------------Navigation--------------------------*/
+.checkbox__popup{
+    appearance: none;
+    background-image: url(./icons/popup-button.png);
+    height: 30px;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 80px;
+    display: block; 
+}
+.checkbox__popup:focus{ 
+    outline: 2px solid rgb(108, 201, 238);
+    border-radius: 3px;
+}
+.nav__lists{
+    display: grid;
+    grid-template-columns: repeat(5,1fr);
+    justify-items: center;
+    align-items: center;
+    
+}  
+         
+.navigation{
+    display: grid;
+    align-items: center;
+    border-radius: 4px;
+    background-color:rgb(246, 247, 239);
+    height: 50px;
+    width: 100%;
+    margin: auto;
+}
+.nav-icons{
+    width: 20px;
+}
+.nav__items{
+    justify-self: center;
+}
+.label__popup{
+    cursor: pointer;
+    display: block;
+}             
+.link,.label__popup{
+    display:grid;
+    align-items: center;
+    justify-items: center;
+    width: 80px;
+    height: 30px;
+}
+.link:hover,.checkbox__popup:hover{
+   outline: 1px solid grey;
+   border-radius: 3px;
+}
+
+/*------------------------popUp----------------------------*/
+.wrap-popup{
+    height: 100%;
+    display:none;
+    width: 100%;
+    justify-items: center;
+    align-items: center;
+}
+
+.checkbox__popup:checked~.wrap-popup,.checkbox__popup:checked~.popup__content-more{
+    display: grid;
+}
+
+.popup__content{
+    display: grid;
+    position: absolute;
+    gap:5px;
+    top: 60px;
+    height: 500px;
+    padding: 10px;
+    background-color:rgb(246, 247, 239);
+    outline: 1px solid black;
+     overflow-y: scroll;
+     box-shadow: 0px 0px 20px grey;
+     border-radius: 3px;
+}
+.icons{
+  width: 40%;
+}
+.popup__lists{
+    display: grid;
+    grid-template-columns: repeat(3,100px);
+    gap:10px;
+    justify-content: center;
+    align-items: center;
+}
+.popup__items{
+    width: 100px;
+    height: 100px;
+}
+.popup__items:hover{
+    outline:1px solid grey;
+    border-radius: 3px;
+}
+  
+a{
+    height: 100%;
+    display: grid;
+    justify-items: center;
+    align-items:center;
+   
+}
+a:focus{
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css). Avoid using too specific CSS selectors. [Read about it.](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)